18 ReactHotLoader/readme.md

```diff
@@ -0,0 +1,120 @@
+# React Hot Loader + Redux dev tool support
```

Comment:
In the about page we can have a summary, but is a good idea to have detailed readme.md when a given user navigates to this sample in Github the readme.md will be automatically displayed